app/client/BUILD

```diff
@@ -0,0 +1,41 @@
+load(
+    "@com_github_grpc_grpc_web//bazel:closure_grpc_web_library.bzl",
+    "closure_grpc_web_library",
+)
+
+load(
+    "@io_bazel_rules_closure//closure:defs.bzl",
+    "closure_js_library",
+    "closure_proto_library",
+    "closure_js_binary"
+)
+
+closure_proto_library(
+  name = "hello_world_service_closure_proto",
+  deps = ["//app/local_server/service:hello_world_service_proto"]
+)
+
+
+# see https://github.com/grpc/grpc-web#wire-format-mode
+closure_grpc_web_library(
+  name = "hello_world_service_grpc",
+  deps = ["//app/local_server/service:hello_world_service_proto"],
+  mode = "grpcweb" # (grpcweb|grpcwebtext)
```

Comment:
`# mode can be grpcweb or grpcwebtext. For more details, see # see https://github.com/grpc/grpc-web#wire-format-mode`